src/main/java/com/videoclub/suriken/api/MovieController.java

```diff
@@ -0,0 +1,52 @@
+package com.videoclub.suriken.api;
+
+import com.videoclub.suriken.model.Movie;
+import com.videoclub.suriken.model.MovieRenter;
+import com.videoclub.suriken.service.impl.MovieServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequestMapping("/api/v1/movie")
+@RestController
+public class MovieController {
+
+    private final MovieServiceImpl movieService;
+
+    @Autowired
+    public MovieController(MovieServiceImpl movieService) {
+        this.movieService = movieService;
+    }
+
+    @PostMapping
+    public void addMovie(@RequestBody Movie movie){
```

Comment:
Msm da ovo ne treba da bude void. Nego kao sto je u onom projektu treba da vratis odredjeni http status kod i neku poruku. Pogledaj moj projekat, pa cemo da diskutujemo

Reply:
Malo su mi mutni ti http response-evi. Analizirao sam kako si ti to odradio, ali moram malo da prodjem kroz to.